packages/server/src/organization/organization.controller.ts

```diff
@@ -808,18 +809,21 @@ export class OrganizationController {
       });
     }
 
-    const fileName =
-      organization.id +
-      '-' +
-      Math.random().toString(36).substring(2, 15) +
-      Math.random().toString(36).substring(2, 15);
+    const fileName = organization.id + '-' + Date.now().toString() + '.webp';
 
-    const image = await Jimp.read(file.buffer);
-    image.resize(1920, 1080);
-    await image.writeAsync(
-      path.join(process.env.UPLOAD_LOCATION as string, fileName),
-    );
-    organization.bannerUrl = fileName;
+    // Create the upload location if it doesn't exist
+    if (!fs.existsSync(process.env.UPLOAD_LOCATION)) {
+      fs.mkdirSync(process.env.UPLOAD_LOCATION, { recursive: true });
+    }
+
+    const targetPath = path.join(process.env.UPLOAD_LOCATION, fileName);
+
+    try {
+      await sharp(file.buffer).resize(1920, 1080).webp().toFile(targetPath);
```

Comment:
I think 1920x1080 might be larger than the org banner needs to be. Since the org banner's height takes up 20% of the screen, I would make the height something like 200 or 300px.